RELEASE_CHANGELOG.md

```diff
@@ -36,94 +36,94 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [v0.7.0] - 2022-11-29
 
-This version base on [lbm-sdk v0.46.0](https://github.com/line/lbm-sdk/releases/tag/v0.46.0).
+This version base on [lbm-sdk v0.46.0](https://github.com/Finschia/finschia-sdk/releases/tag/v0.46.0).
 
 ### Features
-* [\#108](https://github.com/line/lbm/pull/108) Bump github.com/line/lbm-sdk from e19f863a8 to a389b6330
-* [\#110](https://github.com/line/lbm/pull/110) apply GovMint on x/foundation
-* [\#111](https://github.com/line/lbm/pull/111) Bump github.com/line/lbm-sdk from 66988a235 to 0.46.0-rc9
-* (build) [\#113](https://github.com/line/lbm/pull/113) enable to use libsodium version ostracon
+* [\#108](https://github.com/Finschia/finschia/pull/108) Bump github.com/line/lbm-sdk from e19f863a8 to a389b6330
+* [\#110](https://github.com/Finschia/finschia/pull/110) apply GovMint on x/foundation
+* [\#111](https://github.com/Finschia/finschia/pull/111) Bump github.com/line/lbm-sdk from 66988a235 to 0.46.0-rc9
+* (build) [\#113](https://github.com/Finschia/finschia/pull/113) enable to use libsodium version ostracon
```

Comment:
```suggestion
* [\#108](https://github.com/Finschia/finschia/pull/108) Bump github.com/Finschia/finschia-sdk from e19f863a8 to a389b6330
* [\#110](https://github.com/Finschia/finschia/pull/110) apply GovMint on x/foundation
* [\#111](https://github.com/Finschia/finschia/pull/111) Bump github.com/Finschia/finschia-sdk from 66988a235 to 0.46.0-rc9
* (build) [\#113](https://github.com/Finschia/finschia/pull/113) enable to use libsodium version ostracon
```

Reply:
This is commit message, not a url. So we didn't modify it.